java/java.completion/test/unit/data/goldenfiles/org/netbeans/modules/java/completion/JavaCompletionTaskTest/1.8/typesRecordLocalMembersAndVars.pass

```diff
@@ -0,0 +1,111 @@
+boolean
+byte
+char
+double
+final
+float
+int
+long
+short
+X
+Y
+AbstractMethodError
+Appendable
+ArithmeticException
+ArrayIndexOutOfBoundsException
+ArrayStoreException
+AssertionError
+AutoCloseable
+Boolean
+BootstrapMethodError
+Byte
+CharSequence
+Character
+Class
+ClassCastException
+ClassCircularityError
+ClassFormatError
+ClassLoader
+ClassNotFoundException
+ClassValue
+CloneNotSupportedException
+Cloneable
+Comparable
+Compiler
+Deprecated
+Double
+Enum
+EnumConstantNotPresentException
+Error
+Exception
+ExceptionInInitializerError
+Float
+FunctionalInterface
+IllegalAccessError
+IllegalAccessException
+IllegalArgumentException
+IllegalMonitorStateException
+IllegalStateException
+IllegalThreadStateException
+IncompatibleClassChangeError
+IndexOutOfBoundsException
+InheritableThreadLocal
+InstantiationError
+InstantiationException
+Integer
+InternalError
+InterruptedException
+Iterable
+LinkageError
+Long
+Math
+NegativeArraySizeException
+NoClassDefFoundError
+NoSuchFieldError
+NoSuchFieldException
+NoSuchMethodError
+NoSuchMethodException
+NullPointerException
+Number
+NumberFormatException
+Object
+OutOfMemoryError
+Override
+Package
+Process
+ProcessBuilder
+Readable
+Record
```

Comment:
Record will come as we are using it in below test-case
public void testInsideRecParams() throws Exception {
        performTest("Records", 892, null, "typesRecordLocalMembersAndVars.pass", SOURCE_LEVEL);
    }
runtime is jdk-8 but SOURCE_LEVEL is 14 

""(I would also suggest to minimize the number of golden outputs, if/as feasible.)""
--- i will reduce the content of golden files

Reply:
Sorry, but this does not match my understanding of how the tests are working. The content of the golden files is based on the runtime platfom - e.g. the content of java.lang. And when running on JDK 8, there is no java.lang.Record, correct? So when the completion will list all the content of java.lang, it is not clear to me where it would find out j.l.Record?

Reply:
The check was not necessary, removed now
Also golden files were trimmed